Rubberduck.Main/ComClientLibrary/UnitTesting/FakesProvider.cs

```diff
@@ -37,6 +37,30 @@ public void StartTest()
             CodeIsUnderTest = true;
         }
 
+        public static void SuspendFake(String typename)
+        {
+            foreach (var fake in ActiveFakes.Values)
+            {
+                if (fake.GetType().Name == typename)
```

Comment:
...then this becomes `if (fake.GetType() == type)`, no?

Reply:
Perfect, thanks!